.travis.yml

```diff
@@ -54,9 +48,13 @@ install:
   - |
     set -e
     export DEPSPREFIX=$TRAVIS_BUILD_DIR/travisdeps
-    export PREFIX=$TRAVIS_BUILD_DIR/target
-    export PATH=$PATH:$DEPSPREFIX/bin:$PREFIX/bin
-    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$DEPSPREFIX/lib:$PREFIX/lib
+    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
```

Comment:
The cleaner way to do this is to always set the prefix to `/usr` then set the destdir on `make install` to `$TRAVIS_BUILD_DIR/target`. Your packaging script should then be responsible for transforming from that instead of creating the `appdir` folder  up front

Reply:
Using '$TRAVIS_BUILD_DIR/target' as the AppDir now.

Reply:
This wasn't my intent, what I meant was something along the lines of

```sh
export PREFIX=/usr #unconditionally
export TARGET_DIR=$TRAVIS_BUILD_DIR/target
export PATH=$PATH:$DEPSPREFIX/bin:$TARGET_DIR/bin
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$DEPSPREFIX/lib:$TARGET_DIR/lib
```

(effectively replacing PREFIX with TARGET_DIR)

then at make

```
...
make install DESTDIR=$(readlink -f $TARGET_DIR) #unconditionally
``

Reply:
Currently there is

https://github.com/fontforge/fontforge/blob/a05dc286ade51c245f65b591f15dee85dcced172/.travis.yml#L57

I did not want to change that so as to not affect non-Linux builds in any way.

Reply:
I'll take a look at this later today

Reply:
From my experiments, we don't actually need to do the whole prefix/destdir shenanigans, the appimage bundling thingo handles relocating the dlls just fine with the current prefix setup. @probonopd  take a look at the modifications and see what you think